### PR TITLE
[refactor] Movie refactor and add CRUD

### PR DESCRIPTION
**Pull request by meef3k**
In this PR I did:
- CRUD
- refactor
- tests

### CODE REVIEW

File: spec/requests/movies_spec.rb [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      put("/movies/#{movie.id}", params: movie_params)
+      expect(response.status).to eq(302)
+    end
+
+    context 'when params invalid' do
+      let(:movie_duration) { 'duration' }
+
+      it 'returns fail response' do
+        put("/movies/#{movie.id}", params: movie_params)
+        expect(response.status).to eq(302)
```
Comment (baniak66) [inline spec/requests/movies_spec.rb]: You should match the status here with 422 (unprocessable_entity). Please debug that case 😉

**Review by baniak66 — COMMENTED** [review]
Overall LGTM, just one remark 👍

File: spec/requests/movies_spec.rb (hunk 1 above)
Comment (meef3k) [reply spec/requests/movies_spec.rb]: > You should match the status here with 422 (unprocessable_entity). Please debug that case 😉

I'm trying to do that, but I can't parse errors from service to controller

**Review by baniak66 — APPROVED** [review]
🚀

**Pull request merged**